plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/IcebergHiveMetastoreCatalogModule.java

```diff
@@ -40,5 +46,10 @@ protected void setup(Binder binder)
         binder.bind(Key.get(boolean.class, TranslateHiveViews.class)).toInstance(false);
         binder.bind(Key.get(boolean.class, HideDeltaLakeTables.class)).toInstance(HIDE_DELTA_LAKE_TABLES_IN_ICEBERG);
         install(new DecoratedHiveMetastoreModule());
+
+        configBinder(binder).bindConfigDefaults(CachingHiveMetastoreConfig.class, config -> {
+            // disable stats cache by default
+            config.setStatsCacheTtl(new Duration(0, TimeUnit.SECONDS));
```

Comment:
Thanks for caring about Iceberg's specificts.

I think this is redundant, because stats metastore APIs are not used by Iceberg connector at all.

Reply:
> It's not. `io.trino.plugin.iceberg.catalog.MetastoreValidator` checks that caching metastore is off

you're right

so remove change the code comment, as it's not "by default, i.e. can be enabled"
let's have it maybe as `// ensure caching metastore wrapper isn't created, as it's not leveraged by Iceberg`

> if it would make sense to cache stats for Iceberg too.

i am sure it would, but that's orthogonal. The stats don't come from the metastore.